tests/test_coils.py

```diff
@@ -214,9 +287,56 @@ def test_from_symmetry(self):
             coil, I, [0, 0, 1], np.pi / NFP, N // NFP // 2
         )
         coils2 = MixedCoilSet.from_symmetry(coils, NFP, True)
-        B_approx = coils2.compute_magnetic_field([10, 0, 0], basis="rpz", grid=32)[0]
+        B_approx = coils2.compute_magnetic_field(
+            [10, 0, 0], basis="rpz", source_grid=32
+        )[0]
         np.testing.assert_allclose(B_true, B_approx, rtol=1e-3, atol=1e-10)
 
+    @pytest.mark.unit
+    def test_symmetry_magnetic_field(self):
+        """Tests that compute magnetic field is correct from symmetry."""
+        eq = get("precise_QH")
```

Comment:
does this have a non-planar axis? Is that important?

Reply:
Yes it does have a non-planar axis. It shouldn't matter, and if anything it is a more complicated/realistic test case.